src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -64,16 +63,25 @@ public void onMessageReceived(MessageReceivedEvent event) {
             pasteBody = contents;
         }
 
-
         // Create the embed and format it
         EmbedBuilder embedBuilder = new EmbedBuilder();
         embedBuilder.setTitle("Found errors in the log!");
         embedBuilder.setDescription("See below for details and possible fixes");
-        embedBuilder.setColor(0xFF0000);
-
+        embedBuilder.setColor(BotColors.FAILURE.getColor());
 
         List<StackException> exceptions = Parser.parse(pasteBody);
 
+        // Add any errors that aren't from stack traces first
+        for (String issue : TagsManager.getIssueResponses().keySet()) {
+            if (pasteBody.contains(issue)) {
+                addFixIfPresent(issue, embedBuilder);
+            }
+        }
+
+        // Setup 6k limit bounds
+        int limitSize = embedBuilder.getFields().size() / 2;
+        int limitCount = embedBuilder.getFields().size();
+
```

Comment:
Does this need to be calculated *after* we add the non-stacktrace responses? We'll need to also limit the amount of fields added by non-stacktrace responses

Does [this](https://github.com/GeyserMC/GeyserDiscordBot/pull/142/commits/c86ea3081f9d08dcd705d3a657d1410811b3beff#diff-a25460a0da29f69cb8e0c68e9e2979e78d937cd16b71106fe84e604f3d8aba28R95) average only apply to stacktrace stuff?

I don't really understand how `(embedBuilder.getFields().size() - limitCount) + limitSize` works out either. What is `limitCount` suppose to signify?

Reply:
so we can get the total of about 12 stack traces in an embed before we hit the char limit by discord, I use these to make the exception fixes stuff take half the size in the math. Hopefully, that makes sense, not a great impl but best I could think of at the time.